### PR TITLE
chore: add new snapshot service to chain management page

### PR DESCRIPTION
**Pull request by travisperson**
Add a section to the chain management page about the new lightweight snapshot service.

### CODE REVIEW

**Review by ribasushi — CHANGES_REQUESTED** [review]
@travisperson @ognots we **really** should reimplement the `zstd` compression and document it as the preferred way fromt he get go. The snapshot is *huge* today, and zstd takes it down significantly. Specifically at the time of this comment:

```
~$ curl -sI https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car | grep Content-Length
Content-Length: 80751072446
~$ curl -sI https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car.zst | grep Content-Length
Content-Length: 33897687665
```

cc @daviddias who is often bitten by this

**Comment by travisperson** [comment]
Thanks, I've opened an issue to look into it https://github.com/filecoin-project/filecoin-chain-archiver/issues/40

**Comment by ribasushi** [comment]
@travisperson thanks for jumping on this. When implemented: on my client side the typical import looks like:

```
  curl -sL {{urlprefix}}_latest.car.zst \
| zstd -dc \
| lotus daemon --import-snapshot /dev/stdin 2>&1 \
| tee -a ~/daemon-run.log 
```

File: content/en/lotus/manage/chain-management.md [hunk 1]
```diff
@@ -74,6 +74,20 @@ lotus daemon --import-snapshot https://fil-chain-snapshots-fallback.s3.amazonaws
...
+
+**Mainnet**
+```shell
+lotus daemon --import-snapshot https://snapshots.mainnet.filops.net/minimal/latest
+```
+
+**Calibrationnet**
+```shell
+lotus daemon --import-snapshot https://snapshots.calibrationnet.filops.net/minimal/latest
+```
```
Comment (johnnymatthews) [inline content/en/lotus/manage/chain-management.md:89]:
````suggestion
**Mainnet**
```shell
curl -sL https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car.zst | zstd -dc | lotus daemon --import-snapshot /dev/stdin 2>&1 | tee -a ~/daemon-run.log 
```

**Calibrationnet**
```shell
curl -sL https://fil-chain-snapshots-fallback.s3.amazonaws.com/calibrationnet/minimal_finality_stateroots_latest.car.zst | zstd -dc | lotus daemon --import-snapshot /dev/stdin 2>&1 | tee -a ~/daemon-run.log 
```
````

How's this @ribasushi? Haven't tested this though... On Windows right now.

Comment (travisperson) [reply content/en/lotus/manage/chain-management.md:89]: @johnnymatthews this is a new service with different import urls that do not support zst at the moment. @ribasushi comment isn't about this PR, but about supporting zstd in the new serivce. I've filed an issue for it over in the project repository. https://github.com/filecoin-project/filecoin-chain-archiver/issues/40

Comment (johnnymatthews) [reply content/en/lotus/manage/chain-management.md:89]: Ah gotcha. I'll resolve this thread then.

File: content/en/lotus/manage/chain-management.md [hunk 2]
```diff
@@ -74,6 +74,20 @@ lotus daemon --import-snapshot https://fil-chain-snapshots-fallback.s3.amazonaws
 
 {{< /alert >}}
 
+#### New Lightweight Snapshot Service
+
+We have soft launched a new Lightweight chain snapshot service which will be replacing the snapshots above in the future. More information about these snapshots can be found in [Notion](https://pl-strflt.notion.site/Lightweight-Filecoin-Chain-Snapshots-17e4c386f35c44548f5863afb7b5e024).
```
Comment (ognots) [inline content/en/lotus/manage/chain-management.md:79]: add:
This snapshot is still considered experimental, and should not be used for production at this time. 
Please provide feedback in Filecoin Slack in `#fil-infra`

**Review by ognots — CHANGES_REQUESTED** [review]
please see comment

**Comment by ognots** [comment]
@ribasushi @travisperson `zstd` compression is a great idea, but outside the scope/context of this PR.
It should not block merging this PR, and be tracked as an improvement for the next milestone

**Review by ognots — APPROVED** [review]

**Pull request merged**